cypress/support/commands.ts

```diff
@@ -0,0 +1,63 @@
+/// <reference types="cypress" />
+
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace Cypress {
+  interface Chainable {
+    /**
+     * Custom command to login into Atlas with a test membership.
+     */
+    login(): Chainable<void>
+    /**
+     * Custom command to make the polkadot api available and add accounts, memberships, etc.. to it.
+     */
+    injectPolkadot(accounts: any[]): Chainable<void>
```

Comment:
Should probably use types from polkadot: https://github.com/polkadot-js/extension#api-interface

Reply:
Lets do that in another PR to not block the other work as I didn't manage to add it without breaking intellisense right now